includes.mk

```diff
@@ -0,0 +1,11 @@
+check-docker:
+	@if [ -z $$(which docker) ]; then \
+		echo "Missing \`docker\` client which is required for development"; \
+		exit 2; \
+	fi
+
+check-kubectl:
+	@if [ -z $$(which kubectl) ]; then \
```

Comment:
Err... I copied `check-docker` from deis/deis.  `check-kubectl` was copied and modified from `check-docker`.